go-controller/pkg/ovn/base_network_controller_pods.go

```diff
@@ -582,6 +652,11 @@ func (bnc *BaseNetworkController) addLogicalPortToNetwork(pod *kapi.Pod,
 
 	// add external ids
 	lsp.ExternalIDs = map[string]string{"namespace": pod.Namespace, "pod": "true"}
+	if bnc.IsSecondary() {
+		lsp.ExternalIDs[ovntypes.NetworkExternalID] = bnc.GetNetworkName()
```

Comment:
so only if the network is secondary do we add these external ids, right? So if any OVN object does not have these ids, we assume it is primary?

Reply:
yes, we don't existing default network behavior